### PR TITLE
[ES] added content/es/firewall.md

### LINKED ISSUE #1375: [Spanish] localize 'Firewall'
Localize the `content/en/firewall.md` document to Spanish

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
Added `content/es/firewall.md` (translated from `content/en/firewall.md`).
* also removed draft section from the page metadata.

### Related issue number or link (ex: `resolves #issue-number`)
- resolves #1375 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

File: content/es/firewall.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Por ejemplo, en una aplicación bancaria basada en [microservicios](/microservices/), los
+componentes se comunican entre sí transmitiendo información financiera sensible a través de
+la red que comparten. Un actor malicioso podría infiltrarse en esta red, interceptar la
+comunicación y hacer daño si no hubiera un Firewall.
+
+## ¿Cómo ayuda?
+
+Un Firewall examina el tráfico de red utilizando reglas predefinidas.
+Todo el tráfico es filtrado y cualquier comunicación proveniente de una fuente sospechosa o
+no confiable es bloqueada - solo ingresa a la red el tráfico configurado para ser aceptado.
```
[the remark this answers is not in the document]
Comment (marianogg9) [reply content/es/firewall.md]: Agregando un `-` en la linea 27 como comienzo de la oración, la convierte en una linea indentada; lo reemplace con un `;`.

File: content/es/firewall.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: Firewall
+status: Deprecated
+category: Tecnología
```
Comment (electrocucaracha) [inline content/es/firewall.md:4]:
```suggestion
status: Deprecated
draft: true
category: Tecnología
```

File: content/es/firewall.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Problemas que aborda
+
+Por defecto, una red permite a cualquier entidad ingresar y egresar en tanto y en cuanto se
+respeten las reglas de enrutamiento. Por este motivo, es un desafío asegurar una red.
+Por ejemplo, en una aplicación bancaria basada en [microservicios](/microservices/),
+los componentes se comunican entre sí
+transmitiendo información financiera sensible a través de la red que comparten.
+Un actor malicioso podría infiltrarse en esta red, interceptar la
+comunicación y hacer daño si no hubiera un Firewall.
```
Comment (electrocucaracha) [inline content/es/firewall.md:21]:
```suggestion
Por defecto, una red permite a cualquier entidad ingresar y egresar en tanto y en cuanto se respeten las reglas de enrutamiento.
Por este motivo, es un desafío asegurar una red.
Por ejemplo, en una aplicación bancaria basada en [microservicios](/es/microservices/),los componentes se comunican entre sí
transmitiendo información financiera sensible a través de la red que comparten.
Un actor malicioso podría infiltrarse en esta red, interceptar la comunicación y hacer daño si no hubiera un Firewall.
```

File: content/es/firewall.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## ¿Cómo ayuda?
+
+Un Firewall examina el tráfico de red utilizando reglas predefinidas.
+Todo el tráfico es filtrado y cualquier comunicación proveniente de una fuente sospechosa
+o no confiable es bloqueada;
+solo ingresa a la red el tráfico configurado para ser aceptado.
```
Comment (electrocucaracha) [inline content/es/firewall.md:28]:
```suggestion
Todo el tráfico es filtrado y cualquier comunicación proveniente de una fuente sospechosa o no confiable es bloqueada;
solo ingresa a la red el tráfico configurado para ser aceptado.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
Muchas gracias por la traducción @marianogg9. El glosario [utiliza](https://github.com/cncf/glossary/blob/main/content/en/contribute/_index.md?plain=1#L133) [semantic line breaks](https://sembr.org/) para facilitar la lectura de los revisores. Nosotros intentamos mantener esa lógica para facilitar la revisión en la traducción. Es por ello que verás algunas sugerencias en mantener ese formato. Saludos

File: content/es/firewall.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+## ¿Qué es?
+
+Un Firewall es un sistema que filtra tráfico de red basándose en reglas específicas.
+Los Firewall pueden ser hardware, software o una combinación de ambos.
+
+## Problemas que aborda
+
+Por defecto, una red permite a cualquier entidad ingresar y egresar en tanto y en cuanto se respeten las reglas de enrutamiento.
+Por este motivo, es un desafío asegurar una red.
+Por ejemplo, en una aplicación bancaria basada en [microservicios](/microservices/), los componentes se comunican entre sí
```
Comment (electrocucaracha) [inline content/es/firewall.md:18]:
```suggestion
Por ejemplo, en una aplicación bancaria basada en [microservicios](/es/microservices/), los componentes se comunican entre sí
```

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**